rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -720,6 +850,13 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 					continue; // system files are not included in the file list
 				}
 
+				const bool is_media = entry.name == "ICON0.PNG" || entry.name == "ICON1.PAM" || entry.name == "PIC1.PNG" || entry.name == "SND0.AT3";
+
+				if ((is_media && !i) || (!is_media && i))
+				{
+					continue; // add important data files first
```

Comment:
Needs hw testing, it can also be sorted alphabetically in a way that the game also expects it.

Reply:
I know. I kinda hoped someone could investigate

Reply:
any idea on how to sort the stuff though?

Reply:
It seems like the folder is read in order of created/updated which also fits for my game.
Galciv is looking into it.